src/services/__tests__/profileService.test.ts

```diff
@@ -89,13 +89,14 @@ const proxyInitializedProfileResponse = {
   version: 42,
 };
 
-const updateProfileRequest: Profile = {
+const updateProfileRequest: UpdateProfileParams = {
```

Comment:
We need to differentiate between the input for `updateProfile` and `mockUpdateProfile`. The former should not include the new name parameter, whereas the latter should include it.